classes/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rental
```

Comment:
- [ ] As the rental can be attributed to many so `rentals` plurals is the best choice.